src/components/pacbio/PacbioPoolList.vue

```diff
@@ -49,6 +49,6 @@ export default {
 
 .list-group {
   max-height: 400px;
-  overflow: scroll;
+  overflow-y: auto;
```

Comment:
Only shows the scrollbar when needed, and avoids showing the scroll on the x-axis (which isn't needed)